docs/apispec-docs/src/main/scala/sttp/tapir/docs/apispec/schema/ToObjectSchema.scala

```diff
@@ -1,10 +1,11 @@
 package sttp.tapir.docs.apispec.schema
 
-import sttp.tapir.{Codec, Schema => TSchema, SchemaType => TSchemaType}
+import sttp.tapir.SchemaType.SObjectInfo
+import sttp.tapir.{Codec, Validator, Schema => TSchema, SchemaType => TSchemaType}
 
 import scala.collection.mutable.ListBuffer
 
-object ObjectSchema {
+class ToObjectSchema(referenceEnums: SObjectInfo => Boolean = _ => false) {
```

Comment:
same here (default params)

Reply:
also, I think we should move `unique` e.g. to the companion object, as it doesn't need `referenceEnums`, and as far as I remember is called separately (not involved in the `apply` flow)